README.md

```diff
@@ -196,6 +196,23 @@ You can list images in your cluster by running
 
 To add images to the list, simply create `ImageStream` using the above mentioned label
 
+## Spawner API
+
+We are reworking the HTML spawner document to a dynamic spawner page with an API and rendered UI.
+The new API allows gathering data which is shown in the UI.
+The capabilities of the API are:
+  - User ConfigMaps (`GET`, `POST`), It is possible to request the json of the User ConfigMap, and send a body of data to add (or replace). This includes selections, enviroment variables and gpu number.
+  - Sizes (`GET`), Sizes can be requested, either as an array of names (used in the UI) or as a pure json body (using the `pure_json=true` query)
+  - Images (`GET`), Returns an array of images (which have the proper tag). Note: Due to some limitation it is not possible to gather a json with all of the image information
+
+The API also has some primitive local file gathering capability for build purposes.
```

Comment:
What does this mean? What capability?

Reply:
I wanted to mention it because I use the API to access some of the static files (the chunk.js files) and I thought it would be good to mention it if something similar was needed in the future. I never really inteded to use it beyond just accessing the index.html to actually show the page but it could be useful.

Reply:
What you described does not sound to me like what that sentence says:) I would just remove it - it is not extremely important for the actual API